packages/fx-core/tests/core/hooks.test.ts

```diff
@@ -1668,4 +1675,110 @@ describe("Middleware", () => {
       }
     });
   });
+  describe("EnvInfoLoaderMW", () => {
```

Comment:
how about update the UT description to distinguish the status that multi-env is enabled or not?

Reply:
Good point. I forgot that we also need to test cases with multenv disabled.